gtests/net/mptcp/mp_capable/v1_bind_tcpfallback_wrongver_3rd_ack.pkt

```diff
@@ -5,13 +5,18 @@
 +0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
 +0     setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
 
++0     `nstat -n`
 +0     bind(3, ..., ...) = 0
 +0     listen(3, 1) = 0
 +0       <  S   0:0(0)         win 32792  <mss 1000, sackOK, nop, nop, nop, wscale 7, mpcapable v1 flags[flag_h] nokey>
 +0       >  S.  0:0(0)  ack 1             <mss 1460, nop, nop, sackOK, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey]>
 +0.01    <   .  1:1(0)  ack 1  win 257                                               <mpcapable v0 flags[flag_h] key[ckey=2, skey]>
 +0     accept(3, ..., ...) = 4
 
+//  test mibs
++0.01  `test $(nstat -zjs | jq '.kernel.MPTcpExtMPCapableSYNRX') -eq 1`
++0     `test $(nstat -zjs | jq '.kernel.MPTcpExtMPCapableFallbackACK') -eq 1`
+
 // ensure that traffic plane is functional and does not use DSS
```

Comment:
should you eventually add the same test in `dss/mpc_with_data_server.pkt`? We have a SYN + MPC, but no fallback. (even if it is covered by the selftests)

Reply:
so in this case we expect `MPTcpExtMPCapableFallbackACK = 0`  and `MPTcpExtMPCapableSYNRX=1`, correct?

Reply:
Yes, correct, it is just to check this "normal" case.